src/components/DataSetSelect/DataSetSelect.test.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { RootState } from 'reducers';
+import { MemoryRouter } from 'react-router-dom';
+import * as PermissionsUtils from 'utils/PermissionsUtils';
+import { configure, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { Select, Popover } from 'antd';
+import DataSetSelect from './DataSetSelect';
+
+jest.mock('utils/PermissionsUtils');
+
+configure({ adapter: new Adapter() });
+
+const initialStoreState: Partial<RootState> = {
+  datasets: {
+    items: {
+      4448195087284397: {
+        externalId: '693ad162-df1f-408b-87c7-e0ffdaa5e7cf',
+        name: 'Entity Matcher Output',
+        description: 'Made in Data Studio 693ad162-df1f-408b-87c7-e0ffdaa5e7cf',
+        metadata: {
+          COGNITE__SOURCE: 'data-studio',
+        },
+        writeProtected: false,
+        id: 4448195087284397,
+        createdTime: 1576745394155,
+        lastUpdatedTime: 1576745394155,
+      },
+      4610653613010508: {
+        externalId: '4223e48f-57f7-4823-aec1-9a6647512101',
+        name: 'Entity Matcher Output',
+        description: 'Made in Data Studio 4223e48f-57f7-4823-aec1-9a6647512101',
+        metadata: {
+          COGNITE__SOURCE: 'data-studio',
+        },
+        writeProtected: false,
+        id: 4610653613010508,
+        createdTime: 1576745154021,
+        lastUpdatedTime: 1576745154021,
+      },
+    },
+    error: false,
+  },
+};
+
+const store = configureStore([thunk])(initialStoreState);
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+it('can select option', () => {
```

Comment:
Wrap the test suite in one or more levels of `describe` calls

Reply:
yessir